### ISSUE TITLE
[Feature Request] Add support for dashboard overlays

### ISSUE THREAD

**Issue by CrispyPin** [GodotVR/godot_openvr#122]
I am making an overlay app, and in terms of settings and menus I'm restricted to putting it in a regular overlay or on the desktop window. It would make sense to have some settings available from the dashboard, especially meta things like controller bindings.

**Comment by beniwtv** [comment]
Yes, I would like to add that feature, but not sure if it will come here to the Godot OpenVR plugin, or if we'll switch everything over with the OpenXR plugin.

**Comment by TitansWhale** [comment]
Has this feature been completed?
Can OpenXR achieve the dashboard functionality? I didn't see any relevant dashboard interfaces in OpenXR.

> 是的，我想添加该功能，但不确定它是否会出现在 Godot OpenVR 插件中，或者我们是否会使用 OpenXR 插件来切换所有内容。

**Comment by CrispyPin** [comment]
There is an OpenXR extension for overlays, XR_EXTX_overlay, but I don't think SteamVR implements it yet, so nobody is making anything with it. I'm not sure if other OpenXR runtimes like Monado support it.

**Comment by TitansWhale** [comment]
> 有一个 OpenXR 叠加扩展 XR_EXTX_overlay，但我认为 SteamVR 还没有实现它，所以没有人用它做任何事情。我不确定 Monado 等其他 OpenXR 运行时是否支持它。
> 
> [https://steamcommunity.com/app/250820/discussions/8/2448217320142811491/ ](https://steamcommunity.com/app/250820/discussions/8/2448217320142811491/) https://github.com/LunarG/OpenXR-OverlayLayer 似乎你可以使用它在SteamVR中运行OpenXR叠加层，但我还没有尝试过，并且[GodotVR/godot_openxr](https://github.com/GodotVR/godot_openxr)没有提及任何有关 XR_XTX_overlay 的内容。

I agree with your viewpoint. It can be confirmed that SteamVR has not been implemented OpenXR's overlay.
https://github.com/mbucchia/OpenXR-Layer-Template/issues/3
